CHANGELOG.md

```diff
@@ -8,9 +8,10 @@
 
 * **Add Weaviate destination connector** Weaviate connector added to ingest CLI.  Users may now use `unstructured-ingest` to write partitioned data from over 20 data sources (so far) to a Weaviate object collection.
 
-
 ### Fixes
 
+* **Fix Azure destination test directory** Solves a bug where Azure destination test was was looking in the wrong directory.
+
```

Comment:
Sorry, actually coming back to this, I think we typically omit changelog entries for changes that solely affect tests. So maybe revert this part altogether?

Reply:
@ryannikolaidis So I can completely remove changelog and version from this PR?

Reply:
Changelog version still needs to be bumped, but the comment can be removed.